server/middleware/imageUpload.js

```diff
@@ -0,0 +1,73 @@
+import multer from 'multer';
+import dotenv from 'dotenv';
+import mkdirp from 'mkdirp';
+
+dotenv.config();
+
+const cloudinary = require('cloudinary').v2;
```

Comment:
Why are you mixing require and import. Please stick with import.

Reply:
Ok